Cargo.toml

```diff
@@ -25,13 +25,14 @@ version = "0.3.6"
 default-features = false
 
 [dependencies.arrayvec]
-version = "0.5.2"
+# needs to match parity-scale-code which is "=0.7.0"
+version = "0.7.0"
```

Comment:
1. `parity-scale-codec` uses [>=0.7, < 0.8](https://github.com/paritytech/parity-scale-codec/blob/a27e494eabca61e263e43367c9be96b2e0873b0c/Cargo.toml#L12).
2. why does it needs to match `parity-scale-codec`? it's used internally and not exposed in the API
3. why this lib has anything to do with `parity-scale-codec`? cargo allows multiple versions of the same lib (modulo C libs)

Reply:
I ended up publishing this with 0.7.0 but whatever.  We can publish again whenever someone wants an upgrade.